packages/cna-template/template/nuxt/.husky/.gitignore

```diff
@@ -1 +1 @@
-_
\ No newline at end of file
+_
```

Comment:
Part of `lintfix` results thanks to prettier; I think we should integrate it to the template.